package.json

```diff
@@ -102,7 +102,7 @@
     "start-dev": "env-cmd -f .env.dev react-app-rewired start",
     "start-stg": "env-cmd -f .env.stg react-app-rewired start",
     "start-production": "env-cmd -f .env.production react-app-rewired start",
-    "build": "yarn i18n && react-app-rewired build",
+    "build": "yarn i18n && env-cmd -f .env react-app-rewired build",
```

Comment:
@viet-nv need this for `.env` worked.

Reply:
.env is picked up by default, why do you need env-cmd here?

Reply:
Please try it in your local. It doesn't work.

Reply:
yeah. understand. react-script will pick .env file by priority like this: https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used
Thanks

Reply:
This PR won't be merged. It's for testing only.